source/docs/use/diagnostics.md

```diff
@@ -4,13 +4,16 @@ title: Diagnostics
 description: 'How to get HACS diagnostics'
 ---
 
-The diagnostics dump is a JSON file with information about your instance, this is useful to upload to GitHub when creating issues.
-To get the diagnostics dump for HACS do the following:
+The diagnostics file is a JSON file with information about your instance. This file is particularly helpful when you want to report a HACS issue via issue tracker on GitHub. The diagnostics file provides context and helps people understand your setup.
 
-1. First go to the "Settings" in the Home Assistant UI, then "Devices & Services" and the "Integrations" tab in the Home Assistant UI
-1. Then click on the entry for HACS.
-1. Then click on the 3 dots to the right of the card for HACS, and select "Download diagnostics"
+To get the diagnostics file for HACS, follow these steps:
+
+1. In Home Assistant, go to <!-- hacs:my integrations **{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}** -->.
+2. Select the **HACS** integration.
+3. Select the three-dots menu, then select **Download diagnostics**.
+
+    - This will download a file with `json.txt` file ending to the computer you are doing this from (usually the downloads directory), this file can be pasted or dragged into GitHub issues.

```

Comment:
```suggestion
    - This will download a file with `.json` file ending to the computer you are doing this from (usually the downloads directory), this file can be dragged into GitHub issues.

```
- I do not want them to continue to paste the content, uploading the file is the way.
- The filename ending was changed here <https://github.com/home-assistant/core/pull/108028> to just `.json` as GitHub had started allowing that.